zerver/lib/streams.py

```diff
@@ -472,7 +472,7 @@ def check_stream_name_available(realm: Realm, name: str) -> None:
     try:
         get_stream(name, realm)
         raise JsonableError(
-            _("Stream name '{channel_name}' is already taken.").format(channel_name=name)
+            _("Channel name '{channel_name}' is already taken.").format(channel_name=name)
```

Comment:
I think it is very likely worth taking the opportunity (Not necessarily this PR, but during the time before translators redo all the translations) to consider rewriting this one and the `access_stream_by_name` one to be just "Invalid channel name" or "Channel name already in use.", without echoing the actual channel name; the client knows what it requested. Maybe worth an "api design" discussion.

Reply:
It looks like `access_stream_by_name` is used when creating/updating the default stream groups with a list of stream names, so for that case it is probably useful to know what stream name failed.

For `check_stream_name_available`, it's only used in the case of updating a single existing stream's name, so we could probably adjust that one to be "Channel name already in use."

Reply:
Updated the `check_stream_name_available` error message to be "Channel name already in use."

Reply:
> It looks like access_stream_by_name is used when creating/updating the default stream groups with a list of stream names, so for that case it is probably useful to know what stream name failed.

Default stream groups aren't finished as a feature but we can defer this; doesn't seem worth more attention, so marking this one as resolved.